databricks_cli/workspace/cli.py

```diff
@@ -0,0 +1,212 @@
+# Databricks CLI
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"), except
+# that the use of services to which certain application programming
+# interfaces (each, an "API") connect requires that the user first obtain
+# a license for the use of the APIs from Databricks, Inc. ("Databricks"),
+# by creating an account at www.databricks.com and agreeing to either (a)
+# the Community Edition Terms of Service, (b) the Databricks Terms of
+# Service, or (c) another written agreement between Licensee and Databricks
+# for the use of the APIs.
+#
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import click
+from tabulate import tabulate
+from requests.exceptions import HTTPError
+
+from databricks_cli.utils import eat_exceptions, CONTEXT_SETTINGS
+from databricks_cli.version import print_version_callback
+from databricks_cli.configure.config import require_config
+from databricks_cli.dbfs.exceptions import LocalFileExistsException
+from databricks_cli.workspace.api import list_files, mkdirs, import_workspace, export_workspace, \
+    delete, get_status
+from databricks_cli.workspace.types import LanguageClickType, FormatClickType, WorkspaceFormat, \
+    WorkspaceLanguage
+# TODO(andrewmchen): refactor error codes out.
+from databricks_cli.dbfs.api import DbfsErrorCodes
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--absolute', is_flag=True, default=False,
+              help='Displays absolute paths.')
+@click.option('-l', is_flag=True, default=False,
+              help='Displays full information including ObjectType, Language, Path')
+@click.argument('workspace_path', type=str, nargs=-1)
+@require_config
+@eat_exceptions
+def ls_cli(l, absolute, workspace_path):
+    """
+    List files in the Databricks Workspace
+    """
+    if len(workspace_path) == 0:
+        workspace_path = '/'
+    else:
+        workspace_path = workspace_path[0]
+    files = list_files(workspace_path)
+    table = tabulate([f.to_row(is_long_form=l, is_absolute=absolute) for f in files],
+                     tablefmt='plain')
+    click.echo(table)
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.argument('workspace_path')
+@require_config
+@eat_exceptions
+def mkdirs_cli(workspace_path):
+    """
+    Make directories in the Databricks Workspace.
+
+    Note that this function is subtly different than mkdir. Mkdir requires
+    the parent directory of the input to already exist. Mkdirs will create directories
```

Comment:
I think we can skip the first sentence.

Reply:
Mkdirs is clearly different from mkdir. I don't think we need to explain so much. `Mkdirs will create directories along the path to the argument directory.` should be enough.